src/main/java/org/datadog/jenkins/plugins/datadog/clients/HttpClient.java

```diff
@@ -122,7 +126,7 @@ public boolean matches(Origin origin) {
                                     return false;
                                 }
                             }
-                            return true;
+                            return super.matches(origin);
```

Comment:
What cases does this now catch?

Reply:
Requests to proxy host are not proxied :)